File: space triangle/code10.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int a = 0 ; a < i ; a++){

			printf("\t");
		}
	
		for(int j = 0; j < row - i ; j++){

			if(row%2 == 0){
			if( (i+j)%2 == 1 ){

				printf("%c\t",  65 + i  +  j);
			}else{
				printf("%d\t",  65 + i  +  j);
			}
			}else{
			if((i+j)%2 == 0 ){

				printf("%c\t",  65   + i +  j);
			}else{
				printf("%d\t",  65   + i + j);
			}

			}
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/12/code10.java


import java.util.Scanner;

class PyramidDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter rows : ");
		int row = s.nextInt();
		int temp = 1;

		for(int i = 0 ; i < row ; i++){

			for(int k = 0 ; k  < i ; k++){

				System.out.print("		");
			}
			for(int  j = 0 ; j < 2*(row-i-1)+1; j++){
					System.out.print((char)(row+64-i)+"		");
					
				


			}

				System.out.print("\n\n");

		}

	}

}
